.trampolinerc

```diff
@@ -18,8 +18,10 @@ required_envvars+=(
 
 # Add env vars which are passed down into the container here.
 pass_down_envvars+=(
-    "SOURCE_BUCKET"
-    "TEST_BUCKET"
+    "SOURCE_BUCKET"      # Must set to generate any docs.
+    "SOURCE_BLOB"        # Set to force regeneration of a single source blob.
+    "FORCE_GENERATE_ALL" # Set to force regeneration of all blobs.
```

Comment:
Do you need `FORCE_GENERATE_ALL`?

As I read the code, if `SOURCE_BUCKET` is not set, it always generate all the docs.

Reply:
Yes, we need it. If there is no `FORCE_REGENERATE_ALL` and no `SOURCE_BLOB`, `generate.sh` will only generate _new_ docs.

(It's possible there is a bug and I'm missing it.)

Reply:
Ah I see. Now I understand how it works. Thanks.